### PR TITLE
SWL-3777 outer_src_mac, virtual_ip tlvs.

### PR DESCRIPTION
**Pull request by poolakiran**
Reviewer: @adanivas 
CC: @kenchiang @nsong @sriniram

### CODE REVIEW

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_docker/125/

File: openflow_input/bsn_tlv [hunk 1]
```diff
@@ -1113,3 +1113,14 @@ struct of_bsn_tlv_port_speed_gbps : of_bsn_tlv {
...
     uint32_t value;
 };
+
+struct of_bsn_tlv_outer_src_mac : of_bsn_tlv {
+    uint16_t type == 157;
+    uint16_t length;
+    of_mac_addr_t value;
+};
+
+struct of_bsn_tlv_virtual_ip : of_bsn_tlv {
```
Comment (kenchiang) [inline openflow_input/bsn_tlv]: maybe change name to "virtual"

**Comment by bsn-abat** [comment]
Find artifact changes from this pull request at https://github.com/floodlight/loxigen-artifacts/tree/pr-556-jenkins-loxigen_pull_req_artifacts-108

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_artifacts/108/

**Comment by adanivas** [comment]
ACK

**Comment by bsn-abat** [comment]
Find artifact changes from this pull request at https://github.com/floodlight/loxigen-artifacts/tree/pr-556-jenkins-loxigen_pull_req_artifacts-109

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_artifacts/109/

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_docker/126/

**Comment by bsn-abat** [comment]
ABAT: START_MERGE:  Log file at http://vss1/abat/2017.05.19.0051-m.loxigen.master/abat.log

**Comment by bsn-abat** [comment]
ABAT: ACCEPT:  Successfully merged

In case you want to see the build log, check out :
Log file at http://vss1/abat/2017.05.19.0051-m.loxigen.master/abat.log
/cc

**Pull request closed without merging**